autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/imputation/SimpleImputer.py

```diff
@@ -15,92 +13,142 @@
 
 
 class SimpleImputer(BaseImputer):
-    """
-    Impute missing values for categorical columns with '!missing!'
-    (In case of numpy data, the constant value is set to -1, under
-    the assumption that categorical data is fit with an Ordinal Scaler)
+    """An imputer for categorical and numerical columns
+
+    Impute missing values for categorical columns with 'constant_!missing!'
+
+    Note:
+        In case of numpy data, the constant value is set to -1, under the assumption
+        that categorical data is fit with an Ordinal Scaler.
     """
 
-    def __init__(self,
-                 random_state: Optional[Union[np.random.RandomState, int]] = None,
-                 numerical_strategy: str = 'mean',
-                 categorical_strategy: str = 'most_frequent'):
+    def __init__(
+        self,
+        random_state: Optional[Union[np.random.RandomState, int]] = None,
+        numerical_strategy: str = 'mean',
+        categorical_strategy: str = 'most_frequent'
+    ):
+        """
+        Note:
+            Using 'constant' defaults to fill_value of 0 where 'constant_!missing!'
+            uses a fill_value of -1. This behaviour should probably be fixed.
```

Comment:
Is this what you mean?
```suggestion
            'constant' uses fill_value of 0 by default while
            'constant_!missing!' uses fill_value of -1. This behaviour should probably be fixed.
```